app/mailers/identity_mailer.rb

```diff
@@ -1,10 +1,18 @@
 class IdentityMailer < ActionMailer::Base
   default from: "Wavetronix.com <webmaster@wavetronix.com>"
```

Comment:
I recommend changing the name to `Wavetronix Entitlement Server` or something similar, since this email isn't actually coming from the website.